packages/visx-geo/src/projections/Projection.tsx

```diff
@@ -93,6 +93,8 @@ export type ProjectionProps<Datum extends GeoPermissibleObjects = GeoPermissible
   graticuleLines?: Omit<GraticuleProps, 'lines'> & { foreground: boolean };
   /** If specified, renders a Graticule outline with the specified props. Specify `graticuleOutline.foreground = true` to be rendered on top of features. */
   graticuleOutline?: Omit<GraticuleProps, 'outline'> & { foreground: boolean };
+  /** Limits the digits after the decimal to the specified number of digits. */
```

Comment:
Wonder if we could tweak the copy slightly to clarify the digits **of what** (not set on this exactly – open to your thoughts!)
```suggestion
  /** Limits the digits for coordinates generated in SVG path strings to the specified number of digits. */
```